src/utils/fs/collectFilesFromFolder.ts

```diff
@@ -11,7 +11,7 @@ export const collectFilesFromFolder = async (
   maxDepth: number,
   curDepth: number
 ): Promise<WsFiles> => {
-  if (curDepth <= maxDepth) {
+  if (curDepth <= maxDepth || maxDepth === -1) {
```

Comment:
How deep does your folder structure go??? Would a max depth of 100 be OK for your use case or does it really need to go as deep as the folders extend? I'm woried that this could lock up vscode or take a realy long time.